frontend/package.json

```diff
@@ -17,23 +17,47 @@
     "emotion": "^9.1.1",
     "emotion-server": "^9.1.1",
     "html-minifier": "^3.5.14",
+    "jest": "^23.6.0",
```

Comment:
Perhaps jest and ts-jest should be devDependencies?

Reply:
Oh wait in addition I somehow added jest twice. Will tidy this up thanks. 👍